app/src/main/java/org/p2p/wallet/send/model/SendFeeTotal.kt

```diff
@@ -21,14 +20,12 @@ class SendFeeTotal constructor(
     val recipientAddress: String
 ) : Parcelable {
 
-    fun getTotalFee(resourceDelegate: (res: Int) -> String): String =
-        when (sendFee) {
-            is SendSolanaFee ->
-                if (sourceSymbol == sendFee.feePayerSymbol) totalSum
-                else "$totalFormatted + ${sendFee.accountCreationFormattedFee}"
-            else ->
-                resourceDelegate(R.string.send_fees_free)
-        }
+    fun getTotalFee(): String? = when (sendFee) {
+        is SendSolanaFee ->
+            if (sourceSymbol == sendFee.feePayerSymbol) totalSum
+            else "$totalFormatted + ${sendFee.accountCreationFormattedFee}"
+        else -> null
```

Comment:
`SendSolanaFee` is null if user doesn't have a fee for account creation. 
If it's null, we should free transactions info